client/src/main/scala/sbtcompendium/client/CompendiumClient.scala

```diff
@@ -102,7 +106,17 @@ object CompendiumClient {
         } yield out
       }
 
-      override def generateClient(target: IdlName, identifier: String): F[String] = F.pure("")
+      override def generateClient(target: IdlName, identifier: String): F[List[String]] =
+        target match {
+          case IdlName.Avro =>
+            retrieveProtocol(identifier, None)
```

Comment:
It could be in a separate PR, but the protocols to retrieve should be specified here. We'll probably want to tweak the `compendiumProtocolIdentifiers` sbt setting, replacing:

* `compendiumProtocolIdentifiers: SettingKey[Seq[String]]` by `compendiumProtocolIdentifiers: SettingKey[Seq[ProtocolAndVersion]]`

Where `ProtocolAndVersion` could be:

```scala
final case class ProtocolAndVersion(name: String, version: Option[String])
```

.. or something like that. Does it make sense?

Reply:
Yes, I thought the same but I preferred a simpler first version. However, it's ok for me to add it in this PR